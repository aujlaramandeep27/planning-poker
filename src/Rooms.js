import React from 'react';

export default class Rooms extends React.Component {
    
    render() {
        var alphabet = "ABCD";
        return (
            <div className="uk-flex uk-flex-column uk-flex-middle">
                <h1 className="uk-heading-medium uk-margin-top">ROOM</h1>
                { Array.from(alphabet).map(letter => <button className="uk-margin uk-button-secondary uk-button-large uk-width-1-6 uk-text-large">{letter}</button>) }
            </div>
        );
    }
}