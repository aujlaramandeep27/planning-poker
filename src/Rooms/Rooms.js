import React from 'react';

export default function Rooms() {
    
    var alphabets = "ABCD";
    return (
        <div className="uk-flex uk-flex-column uk-flex-middle">
            <h1 className="uk-heading-medium uk-margin-top">ROOM</h1>
            { Array.from(alphabets).map((letter,i) => 
                <button key={i} className="uk-margin uk-button-secondary uk-button-large uk-width-1-6 uk-text-large">{letter}</button>)
            }
        </div>
    );
}