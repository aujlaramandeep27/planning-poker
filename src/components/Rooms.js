import React from 'react';
import StackedLinks from './StackedLinks';

export default function Rooms() {
  const rooms = [
    { 'href': '/room/A', 'label': 'A' },
    { 'href': '/room/B', 'label': 'B' },
    { 'href': '/room/C', 'label': 'C' },
  ];

  return (
    <>
      <h1>Rooms</h1>
      <div className="uk-container">
        <StackedLinks links={rooms} />
      </div>
    </>
  );
}
