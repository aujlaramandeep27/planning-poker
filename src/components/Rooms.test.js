import Rooms from './Rooms';

describe('Component: Rooms', () => {
  test('should render', () => {
    expect(Rooms()).toMatchSnapshot();
  });
});
