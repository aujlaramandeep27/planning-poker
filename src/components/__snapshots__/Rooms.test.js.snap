// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Rooms should render 1`] = `
<React.Fragment>
  <h1>
    Rooms
  </h1>
  <div
    className="uk-container"
  >
    <StackedLinks
      links={
        Array [
          Object {
            "href": "/room/A",
            "label": "A",
          },
          Object {
            "href": "/room/B",
            "label": "B",
          },
          Object {
            "href": "/room/C",
            "label": "C",
          },
        ]
      }
    />
  </div>
</React.Fragment>
`;
